docs/readme.md

```diff
@@ -1,23 +1,25 @@
-# Project Overview
+# Getting started
 
-The Gutenberg project is transforming the way content is created on WordPress. A block editor was the first product launched creating a new methodology for working with content.
+"Gutenberg" is the codename for the new WordPress editor, also known as the block editor. The Gutenberg project is transforming the way content is created on WordPress.
 
-The Block Editor handbook provides documentation for designers and developers on how to extend the editor, and also how you can start contributing to the project. For authors, writers, and users of the block editor see the [block editor support documentation](https://wordpress.org/support/article/wordpress-editor/).
-
-![Gutenberg Demo](https://cldup.com/kZXGDcGPMU.gif)
+![Quick view of the block editor](https://make.wordpress.org/core/files/2021/01/quick-view-gutenberg-editor.png)
```

Comment:
This feels like an outdated screenshot. I'd love if we could get something a bit snazzier in place. If not, no worries. Definitely should link to this though here perhaps: https://wordpress.org/gutenberg/

Reply:
Actually I made the screenshot this week.
But as Marcus said (https://github.com/WordPress/gutenberg/pull/28142#pullrequestreview-567490657), it could use some improvements. I'll make sure to have the feedback of the design team about it on Tuesday.